Slider/Assets/Prefabs/Player/Player.prefab

```diff
@@ -168,7 +168,7 @@ MonoBehaviour:
     ModeOverride: 0
     LensShift: {x: 0, y: 0}
     GateFit: 2
-    m_SensorSize: {x: 1.7777779, y: 1}
```

Comment:
do you remember what you changed about player prefab

Reply:
Added a new drop mask for anchor vs non-anchor items so u cant throw non-anchor items onto crates